src/test/scala/edu/ie3/simona/model/participant/BMModelSpec.scala

```diff
@@ -174,7 +174,7 @@ class BMModelSpec extends UnitSpec {
       )
 
       val pElCalc = bmModel.calculateElOutput(usage, eff)
-      pElCalc.value should be(Kilowatts(pElSol).value +- 1e-4)
+      pElCalc.value should be(Kilovoltamperes(pElSol).toVoltamperes +- 1e-4)
```

Comment:
Should we check here for the unit as well?

Reply:
We can check for the unit here. Should I add these check to other tests as well?

Reply:
Yes, please. That would be great

Reply:
After looking at other tests, I found that using the `approximate` method will check the value as well as the unit.

Reply:
Yes that was the idea. Sorry, could have written that directly. Thanks for adapting it!